core/Tracker/Cache.php

```diff
@@ -119,6 +119,14 @@ public static function clearCacheGeneral()
     }
 
     /**
+     * Clear website cache.
+     */
+    public static function clearWebsiteCache($idSite)
+    {
+        self::getCache()->delete((int) $idSite);
+    }
```

Comment:
CustomDimensions store attributes for a site in Tracker Cache. For best user experience we clear the cache when dimensions change for a website.